src/main/java/ch/njol/skript/effects/EffStopSound.java

```diff
@@ -36,97 +28,113 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
+import org.bukkit.Sound;
+import org.bukkit.SoundCategory;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.Locale;
+import java.util.regex.Pattern;
 
 @Name("Stop Sound")
-@Description({"Stops a sound from playing to the specified players. Both Minecraft sound names and " +
+@Description({"Stops the given sound(s) from playing to the specified players. Both Minecraft sound names and " +
 		"<a href=\"https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Sound.html\">Spigot sound names</a> " +
 		"are supported. Resource pack sounds are supported too. The sound category is 'master' by default. " +
 		"A sound can't be stopped from a different category. ",
 		"",
 		"Please note that sound names can get changed in any Minecraft or Spigot version, or even removed from Minecraft itself."})
```

Comment:
```suggestion
@Description({
	"Stops specific or all sounds from playing to a group of players. Both Minecraft sound names and " +
	"<a href=\"https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Sound.html\">Spigot sound names</a> " +
	"are supported. Resource pack sounds are supported too. The sound category is 'master' by default. " +
	"A sound can't be stopped from a different category. ",
	"",
	"Please note that sound names can get changed in any Minecraft or Spigot version, or even removed from Minecraft itself."
})
```
I think this reads better (plus formatting fixes)